### ISSUE TITLE
Emulating backspace key in Handy Tech Braille Displays

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3993]
**Reported by ateu on 2014-03-19 02:03**
Received from a Portuguese member.

When typing in a Braille keyboard in a Handy Tech Display, it's not practical to delete something, as NVDA cannot emulate the backspace key, so that the user needs to move the hand from the braille keyboard to the computer to press the backspace key.
For this reason, it would be nicer if this key can be assigned.

**Comment by leonardder** [comment]
Note that Handy Tech braille input isn't supported by NVDA core, only by the unofficial custom add-on from Handy Tech themselves. However, backspace is properly mapped to the b1 key here, which corresponds to dot seven on a braille keyboard. Since this issue only talks about the backspace and not about the lack of braille input functionality, closing as works for me.

CC @FelixGruetzmacher
